docs/tutorials/add-prisma-database.md

```diff
@@ -0,0 +1,344 @@
+
+Prisma is an open-source toolkit that simplifies database access.
+You can always fall back to the official docs for deeper dives: https://www.prisma.io/docs/.
+
+```{note}
+The main purpose of this tutorial is to demonstrate how to integrate Prisma with Seven.
+You can also use the same approach but saving to a Plone REST API endpoint.
+It is not meant to be a complete, production-ready implementation of a like button.
+You can find the complete code of this tutorial in this [GitHub repository](https://github.com/collective/seven-training-addon).
+```
+
+## Prerequisites
+
+- A Seven add-on scaffolded with {term}`cookieplone`
+- {term}`pnpm` installed and configured for the workspace
+- Basic familiarity with TypeScript/React Router code in a Seven project
+
+If you still need an add-on skeleton, follow {doc}`../get-started/create-package` before you continue.
+
+## Install Prisma in your add-on
+
+Run this command from the repo root to install Prisma and its client in your add-on package:
+
+```shell
+pnpm --filter <addon-name> add @prisma/client prisma @types/node
+```
+
+```{warning}
+{term}`pnpm` will require you to approve the builds of this packages, by running `pnpm approve-builds` command.
+Follow the instructions in the terminal to approve the builds.
+
+```{image} ../_static/approve-builds.jpg
+:alt: pnpm approve-builds
+:width: 700px
+:align: center
+```
+
+In your add-on's `package/<addon-name>/package.json`, add these convenience scripts to work with the Prisma client:
+
+```{code-block} json
+:caption: packages/\<addon-name>/package.json
+"scripts": {
+    // Other scripts...
+    "prisma:generate": "prisma generate",
+    "prisma:db:push": "prisma db push",
+    "prisma:migrate": "prisma migrate dev",
+    "prisma:studio": "prisma studio"
+}
+```
+
+Keep any existing scripts you already have in place—only add the Prisma entries.
+
+## Set up the Prisma schema
+
+Create `packages/<addon-name>/prisma/schema.prisma` with the data model that will hold each page's like counter:
+
+```{code-block}
+:caption: packages/\<addon-name>/prisma/schema.prisma
+generator client {
+  provider = "prisma-client"
+  output   = "../generated/prisma"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model UrlLike {
+  id        Int      @id @default(autoincrement())
+  pathname  String   @unique
+  count     Int      @default(0)
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+}
+```
+
+This schema configures Prisma to generate its client into `packages/<addon-name>/generated/prisma` and to store data in SQLite by default. Replace the provider if you want to target PostgreSQL, MySQL, MongoDB, or any other Prisma-supported database in production.
+
+The URL of the database is read from the `DATABASE_URL` environment variable.
+
+The development database will use sqlite for simplicity and it will be stored in a file named `dev.db` in your add-on folder `packages/<addon-name>/prisma/dev.db`.
```

Comment:
```suggestion
The development database will use SQLite for simplicity.
It will be stored in a file named {file}`dev.db` in your add-on folder {file}`packages/<add-on-name>/prisma`.
```